authors/apps/articles/tests/test_article_report.py

```diff
@@ -0,0 +1,73 @@
+from .base import BaseTest
+from rest_framework.views import status
+
+
+class ReportArticleTestCase(BaseTest):
+    def get_token(self):
+        response = self.client.post(self.SIGN_UP_URL, self.fav_test_user, format='json')
+        response = self.client.post(self.SIGN_IN_URL, self.fav_test_user, format='json')
+        token = response.data['token']
+        return token
+
+    def create_article(self, token, article):
+        """ Method to create an article"""
```

Comment:
also in base.py there's an article posted already with the slug test-title you can make use of that too.